### PR TITLE
Manual Workflow: test wheels from Azure build

### PR DESCRIPTION
**Pull request by alexsavulescu**
* feature to test wheels from Azure build (PR, master, ...)

### CODE REVIEW

**Comment by alexsavulescu** [comment]
**cc**: @pramodk

File: scripts/install_redhat_centos:7.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 yum install -y epel-release centos-release-scl centos-release-scl-rh
 
 # Install a newer toolchain for CentOS7
-yum install -y cmake3 ${SOFTWARE_COLLECTIONS_centos_7}
+yum install -y cmake3 ${SOFTWARE_COLLECTIONS_centos_7} wget unzip
```
Comment (olupton) [inline scripts/install_redhat_centos:7.sh:7]: I don't think you need this given that you added these to `install_redhat.sh` too.

File: scripts/install_redhat_centos:8.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # Enable the (official) PowerTools repository. This provides Ninja.
-dnf install -y dnf-plugins-core
+dnf install -y dnf-plugins-core wget unzip
```
Comment (olupton) [inline scripts/install_redhat_centos:8.sh:3]: I don't think you need this given that you added these to `install_redhat.sh` too.

**Review by olupton — APPROVED** [review]
LGTM, just a couple of minor nitpicks.

File: scripts/install_redhat_centos:7.sh (hunk 1 above)
Comment (alexsavulescu) [reply scripts/install_redhat_centos:7.sh:7]: I will check, thanks

**Pull request merged**